sonic_data_client/virtual_db.go

```diff
@@ -145,7 +149,7 @@ func getPfcwdMap() (map[string]map[string]string, error) {
 		}
 
 		for _, key := range resp {
-			name := key[7:]
+			name := key[13:]
```

Comment:
>13 [](http://example.com/codeflow?start=15&length=2)

Where is 13 from? Add some code comment?

Reply:
Seems like Zain recently fixed a bug, but you branch is up-to-date. Could you rebase or merge master?

Reply:
Fixed